README.md

```diff
@@ -1,104 +1,159 @@
 
-```bash
-yarn install
-yarn test
-yarn build
+We generate a Cohort object by:
+
+```javascript
+import { Cohort } from '@nucypher/nucypher-ts';
+
+const config = {
+  threshold: 3,
+  shares: 5,
+  porterUri: 'https://porter-lynx.nucypher.community',
+};
+const newCohort = await Cohort.create(config);
 ```
 
-## Development
+Notice that we have also provided a `porterUri`. **Porter is a web-based service that interacts with nodes on the network on behalf of applications – an "Infura for TAC".** In this example, we've chosen a `lynx` (testnet) Porter endpoint.
 
-Install git hooks
+## 3. Create Conditions
 
-```bash
-npx husky install
+We will now specify the conditions on which data access will be predicated – i.e. what will the data requester need to prove in order to gain decryption rights. In this tutorial, nodes will check that the requester owns a specific ERC721 NFT:
+
+```javascript
+import { Conditions } from '@nucypher/nucypher-ts';
+
+const NFTOwnership = new Conditions.ERC721Ownership({
+  contractAddress: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D',
+  chain: 5, // lynx uses the Goerli testnet
+  parameters: [5954],
+});
 ```
 
-Generate contract typings
+Note that there are other Condition [types](https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions), and it is possible to compose and combine multiple Condition objects into a [_ConditionSet_](https://docs.threshold.network/app-development/threshold-access-control-tac/references/condition-set)_:_
 
-```bash
-yarn typechain
+```javascript
+import { Conditions, ConditionSet } from '@nucypher/nucypher-ts';
+
+const conditions = new ConditionSet([NFTOwnership]);
 ```
 
-Prepare a new release
+In this tutorial, we'll only specify a single Condition to access the data.
 
-```bash
-yarn run prepare-release
+## 5. Build a Strategy
+
+We now bundle the Cohort, ConditionSet, and any other extra parameters into a [_Strategy_](https://docs.threshold.network/app-development/threshold-access-control-tac/references/strategy)_:_
+
+```javascript
+import { Strategy } from '@nucypher/nucypher-ts';
+
+const newStrategy = Strategy.create(
+  newCohort,
+  conditions
+);
 ```
 
-## Publishing
+Next, we deploy this Strategy to the Threshold Network:
 
-Publish a new release on NPM.
+```typescript
+import detectEthereumProvider from '@metamask/detect-provider';
+import providers from 'ethers';
 
-Pay attention to output of these commands and fix your release if needed.
+const MMprovider = await detectEthereumProvider();
+const goerli = providers.providers.getNetwork('Goerli');
 
-To build and publish a release, run
+if (MMprovider) {
+  const web3Provider = new providers.providers.Web3Provider(
+    MMprovider,
+    goerli
```

Comment:
Does Goerli have `SubscriptionManager` contract deployed? Has this code been tested (or ran) recently?

Reply:
that's a good point, payments are actually turned off for goerli at the moment. I will add some wording that explains this

Reply:
Since it's meant to target L2 for affordable subscriptions `SubscriptionManager` is only deployed to Polygon Mainnet and Polygon/Mumbai.